contracts/interfaces/ITwabRewards.sol

```diff
@@ -15,17 +15,17 @@ interface ITwabRewards {
         @param creator Addresss of the promotion creator
         @param ticket Prize Pool ticket address for which the promotion has been created
         @param token Address of the token to be distributed as reward
-        @param tokensPerEpoch Number of tokens to be distributed per epoch
         @param startTimestamp Timestamp at which the promotion starts
+        @param tokensPerEpoch Number of tokens to be distributed per epoch
         @param epochDuration Duration of one epoch in seconds
         @param numberOfEpochs Number of epochs the promotion will last for
      */
     struct Promotion {
```

Comment:
I think this can be better... I commented the details in another PR.

Basically, you can use bigger datatypes for the timestamps and still fit everything in 3 words

Reply:
I've updated the datatypes, I will reorder the variables in the following PR: https://github.com/pooltogether/v4-periphery/pull/47